Document/0x05g-Testing-Network-Communication.md

```diff
@@ -117,6 +117,19 @@ Certificate pinning is the process of associating the backend server with a part
 
 The certificate can be pinned and hardcoded into the app or retrieved at the time the app first connects to the backend. In the latter case, the certificate is associated with ("pinned" to) the host when the host is seen for the first time. This alternative is less secure because attackers intercepting the initial connection can inject their own certificates.
 
+Note that there are various ways to work with a failing pin:
+
+- Inform the user about not being able to connect to the back-end and stop all operations. The app can check whether there is an update and inform the user about updating to the latest version of the app if available.
```

Comment:
style guide "back end"
```suggestion
- Inform the user about not being able to connect to the back end and stop all operations. The app can check whether there is an update and inform the user about updating to the latest version of the app if available.
```

Reply:
This sounds like what "Pokémon Go" does. Once there's an update, they don't care that you're in the middle of nowhere. The app refuse to start, and redirects to the Play store so you can click on "Update" (and maybe you have to wait until the next Wi-Fi connection). So no Pokémons until you reach home, and that's ok for them.

**-> Should we highlight here that the app refuses to start completely? Maybe put the redirection to the Play Store as an example of how the app reacts?**

Reply:
I guess not. Developers and customers that have had this instructions before from me understood it. Lets try to become less verbose and to the point :-)